WalletWasabi/WabiSabi/Client/AmountDecomposer.cs

```diff
@@ -40,7 +44,6 @@ public AmountDecomposer(FeeRate feeRate, MoneyRange allowedOutputAmount, int ava
 
 	public IOrderedEnumerable<Output> Denominations { get; }
 	public ScriptType ChangeScriptType { get; }
-	public Money ChangeFee => FeeRate.GetFee(ChangeScriptType.EstimateOutputVsize());
```

Comment:
This is now unnecessary. I am sure there are some insignificant gains to be made by adding complexity with it. I don't care about that.

If the removal of this results in a bug. Then please point out. Leave the optimization away from this crucial code.